src/util.c

```diff
@@ -705,3 +706,29 @@ mark_data_as_dirty(NetplanParser* npp, void* data_ptr)
         npp->current.netdef->_private->dirty_fields = g_hash_table_new(g_direct_hash, g_direct_equal);
     g_hash_table_insert(npp->current.netdef->_private->dirty_fields, data_ptr, data_ptr);
 }
+
+/**
+ * Copy a NUL-terminated string into a sized buffer, and returns the size of
+ * the copied string, including the final NUL characted. If the buffer is too
```

Comment:
```suggestion
 * the copied string, including the final NUL character. If the buffer is too
```
nitpick (non-blocking): typo

Reply:
Fixed, thanks!